scripts/modules/b2b-account/quotes.js

```diff
@@ -251,6 +255,13 @@ define([
                 }
             }
         },
+        callbackForQuoteGridSelection:function(data){
```

Comment:
Ideally this should be called `selectRowInQuotesGrid` or `quotesGridRowSelected` as it better describes the action.

Reply:
I have made it `quotesGridRowSelected`.